diagnostics.yml

```diff
@@ -193,7 +194,7 @@ stages:
         parameters:
           name: Linux_arm
           osGroup: Linux
-          dockerImage: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-20210719121212-8a8d3be
+          dockerImage: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross
```

Comment:
I think we want to update to the 18.04 images like @michellemcdaniel  did in her PR: https://github.com/dotnet/diagnostics/pull/3492.  This comment applies to all of the other ubuntu 16.04 images and that PR removes the 16.04 build leg since it is out of service now.

Reply:
Yeah, there 16.04 is out of service, so those images don't exist anymore. 18.04 is probably easiest to move to.